roles/splunk_common/tasks/remove_clustering.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Remove indexer clustering stanza in order to reconfigure
+  ini_file:
+    dest: "{{ splunk.home }}/etc/system/local/server.conf"
+    section: "clustering"
+    state: absent
+    owner: "{{ splunk.user }}"
+  become: yes
+  become_user: "{{ splunk.user }}"
+
+- name: Remove search head clustering stanza in order to reconfigure
```

Comment:
Looks like this gets triggered when `splunk_indexer_cluster` which implies there's a cluster master/indexer cluster. But that doesn't necessarily mean there's also a SHC.

Reply:
Yes, it will clear the stanzas only if they exist. It is a no-op if they don't exist.